src/plugins/__mocks__/database.ts

```diff
@@ -1,6 +1,8 @@
-import { FastifyPluginAsync } from 'fastify';
 import { createMockPool, createMockQueryResult } from 'slonik';
-import { Database, DatabasePluginOptions } from '../database';
+
+import { FastifyPluginAsync } from 'fastify';
+import {Database} from '@graasp/sdk';
```

Comment:
I think you are missing some spaces here 
```suggestion
import { Database } from '@graasp/sdk';
```